server/src/main/java/org/apache/uniffle/server/ShuffleServerConf.java

```diff
@@ -500,8 +500,11 @@ public class ShuffleServerConf extends RssBaseConf {
   public static final ConfigOption<Integer> NETTY_SERVER_WORKER_THREAD =
       ConfigOptions.key("rss.server.netty.worker.thread")
           .intType()
-          .defaultValue(100)
-          .withDescription("Worker thread count in netty");
+          .defaultValue(0)
```

Comment:
Could you refer to the Spark similar implement? Does Spark use `0` or nodefault or other value?

Reply:
> Could you refer to the Spark similar implement? Does Spark use `0`, `-1` or nodefault or other value?

In Spark, there isn't a standard for default configuration values; some use `nil`, some use `0`. I think using 0 in our case is similar to the scenario of configuring the `spark.blockManager.port`:
The default value of `spark.blockManager.port` is set to 0, which indicates a random free port.

Reply:
You should refer to the thread number of Spark.

Reply:
> You should refer to the thread number of Spark.

Refer to `org.apache.spark.network.util.TransportConf`, the default value is also `0`
(In https://github.com/apache/spark/blob/master/common/network-common/src/main/java/org/apache/spark/network/util/TransportConf.java#L130):
```
public class TransportConf {
  ...
  /** Number of threads used in the server thread pool. Default to 0, which is 2x#cores. */
  public int serverThreads() { return conf.getInt(SPARK_NETWORK_IO_SERVERTHREADS_KEY, 0); }
  ...
}
```